measure_extinction/extdata.py

```diff
@@ -126,6 +126,28 @@ def _get_column_val(column):
         return float(column)
 
 
+def _get_column_plus_unc(column):
+    """
+
+    Parameters
+    ----------
+    column : float or tuple
+        gives the column or (column, unc) or (column, punc, munc)
```

Comment:
Throughout the extdata code it is punc, munc.  The the answer is yes, I'm usually doing the opposite.

Reply:
Then I am afraid there might be a conflict in the save function of extdata.py. For the columns, it is taking [1] as the minus and [2] as the plus uncertainty...

https://github.com/karllark/measure_extinction/blob/d13198cac9b870f07c8da36b095910f3cb2d23ab/measure_extinction/extdata.py#L765-L792

Reply:
This is another example why it would be better to have an astropy table instead of keywords in the header, see issue #77.

Reply:
I'm going to change the save code to be punc in 1 and munc in 2 so that at least the code is consistent.  I agree, it would be good to do this via a table instead of keywords.